native/templates/openharmony/entry/src/main/ets/MainAbility/cocos/jsb-adapter/sys-ability-polyfill.js

```diff
@@ -0,0 +1,21 @@
+import display from '@ohos.display';
```

Comment:
Should use `_` in file name.

Reply:
we follow the import import standard in their docs https://developer.harmonyos.com/cn/docs/documentation/doc-references/js-apis-display-0000001281001106

I'm not sure if it can work when we import like this 
```ts
import _display from '@ohos.display';
```

Reply:
we name file following the format `xxx-xxx.js`   like `jsb-engine.js`